src/App.css

```
@@ -1 +1,14 @@
+.product {
+    display: grid;
+    grid-template-columns: repeat(2,1fr);
+}
 
+.products {
+    display: grid;
+    grid-template-columns: repeat(3,1fr);
+}
```

```suggestion
.products {
    display: grid;
    grid-template-columns: repeat(2,1fr);
}
```
Based on the example, the products grid should actually be two columns, not three.